README.md

```diff
@@ -17,8 +17,8 @@ To install this library in your `GOPATH`:
 
 ```
 mkdir -p $GOPATH/src/github.com/kolide/
```

Comment:
What about this line? And do we even still need gopath?

Reply:
GOPATH should no longer be necessary. I updated this README to indicate that it installs like normal, however I'm not actually sure that's the case until we merge the change (since I can't test it due to the issue this PR attempts to resolve).